shell/public/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="">
+
+<head>
+    <meta charset="utf-8">
```

Comment:
In the old world build in packages (pkgs in `/pkg` that were `autoLoad`ed via `autoLoadPackages` in nuxt.config were brought in here. `autoLoadPackages` does exist in vue.config.js but isn't used anywhere. Does this feature still work?

Reply:
What gives you the impression autoLoadPackages was used here before? Can you link to an example because 
it looks like it's not used anywhere https://github.com/rancher/dashboard/search?q=autoLoadPackages&type=

This was just copied over as part of the initial conversion.

Reply:
Update to follow in main review comment, but this is applied with https://github.com/rancher/dashboard/blob/master/shell/nuxt.config.js#L588